### PR TITLE
docs/esp8266/general: nit-picking

### PR DESCRIPTION
**Pull request by mad474**
Very first PR: Please shout at me if it's not the right time (release rush), method or place for contributing (and learning. Trying to find something useful to do while learning micropython+git+github).

### CODE REVIEW

File: docs/esp8266/general.rst [hunk 1]
```diff
@@ -18,8 +18,8 @@ To make a generic ESP8266 port and support as many boards as possible,
 following design and implementation decision were made:
 
 * GPIO pin numbering is based on ESP8266 chip numbering, not some "logical"
-  numbering of a particular board. Please have manual/pin diagram of your board
-  handy to find correspondce between your board pins and actual ESP8266 pins.
+  numbering of a particular board. Please have the manual/pin diagram of your board
+  handy to find diversions between your board pins and actual ESP8266 pins.
```
Comment (dhylands) [inline docs/esp8266/general.rst]: Actually, diversions isn't right. I'd either stick with diversions or perhaps use "the mapping".

**Comment by dhylands** [comment]
I'd recommend that you squash all of your commits together (since they're logically all doing the same thing):
https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

In the future, you can use git commit --amend to adjust your latest commit (rather than creating a new one). You don't need to create a new PR. github is branch based, so you can do a forced push and this will update the PR automagically. I normally then go to github and leave a comment that I've done that since the forced push doesn't trigger any notifications.

**Comment by mad474** [comment]
Thanks Dave! Closing, furthering studies, retrying subsequently.

**Comment by pfalcon** [comment]
Reopening, I'd like to pick up articles from with PR.

**Comment by mad474** [comment]
:) Thanks for encouragement Paul! Because of daywork distraction it may take another 3 weeks before I stop producing git/github ridiculousness. Anyway, aiming to contribute in documentation cases (which also helps me to learn python) - far aim is german translation - maybe beginning coming winter if nobody else precedes ;)

**Comment by pfalcon** [comment]
Thanks, finally merged, just reverting "diversions" word, as it's definitely not what I meant ;-).

**Pull request closed without merging**